Kitodo/src/test/java/org/kitodo/production/helper/tasks/MigrationTaskIT.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.helper.tasks;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.kitodo.MockDatabase;
+import org.kitodo.data.database.beans.Project;
+import org.kitodo.production.services.ServiceManager;
+
+public class MigrationTaskIT {
+
+    private static Project project;
+
+    @BeforeClass
+    public static void prepareDatabase() throws Exception {
+        MockDatabase.startNode();
+        MockDatabase.insertProcessesFull();
+
+        project = ServiceManager.getProjectService().getById(1);
+    }
+
+    @AfterClass
+    public static void cleanDatabase() throws Exception {
+        MockDatabase.stopNode();
+        MockDatabase.cleanDatabase();
+    }
+
+    @Test
+    public void testMigrationTask() throws InterruptedException {
```

Comment:
test should test, if metadata is migrated ;-)

Reply:
This is covered by MetsKitodoConverterTest (in Kitodo-DataEditor module)

Reply:
More specific: The test should test, if the metadata migration is actually triggered by the task. if I remove everything out of the run() method (but the setProgress100) the test is still green.

Reply:
can't you reuse the test logic from the converter test? A single migration should be enough to test, if everything is triggered alright